swarm/network/stream/delivery.go

```diff
@@ -131,7 +131,7 @@ func (s *SwarmChunkServer) GetData(key []byte) ([]byte, error) {
 
 // RetrieveRequestMsg is the protocol msg for chunk retrieve requests
 type RetrieveRequestMsg struct {
-	Key       storage.Key
+	Key       storage.Address
```

Comment:
Renaming field `Key` to `Address` will break backward compatibility. If we do not care for it currently, this should be also renamed. Like in other message types in this package.

Reply:
i dont think it necessarily break anything actually. we use RLP encoding using reflect
if reflect's field order is unchanged, there is no problem

Reply:
That is great. 👍